pkg/data/values.go

```diff
@@ -18,18 +27,43 @@ func GetValueN(data map[string]interface{}, keys ...string) interface{} {
 	return val
 }
 
-func GetValue(data map[string]interface{}, keys ...string) (interface{}, bool) {
+// GetValue gets a value from the provided data field, which must be a map[string]interface or a []interface. Keys can
```

Comment:
I would tweak the doc comment like so:

```
// GetValue retrieves a value from the provided collection], which must be a map[string]interface or []interface. 
// Keys are always strings. 
// For the map, they denote the path to the value within the nested structure of the map.
// For the slice, they denote the index.
// Returns the retrieved value (if any) and a bool indicating if they value was found.
```

Reply:
I've made some adjustments to the doc comment. LMK if you want any further changes.